classes/notification/PKPNotification.inc.php

```diff
@@ -62,15 +62,13 @@
 define('NOTIFICATION_TYPE_PENDING_INTERNAL_REVISIONS',		0x1000015);
 define('NOTIFICATION_TYPE_PENDING_EXTERNAL_REVISIONS',		0x1000016);
 define('NOTIFICATION_TYPE_COPYEDIT_ASSIGNMENT',			0x1000017);
-define('NOTIFICATION_TYPE_ALL_REVIEWS_IN',			0x1000018);
```

Comment:
Ok, I've done that. Tested with the upgrade script and seemed to work. But can you take a look since this is the first time I've done something like this:

https://github.com/NateWr/ojs/commit/d6f909bf7e8e77e42d37b1960bd6401f6e7281c3
https://github.com/NateWr/omp/commit/30b1d84282c738b3f5bcecf0e5854c83ce7c971b